sbc_translation/data/nas_parameters.xml

```diff
@@ -0,0 +1,13 @@
+<openerp>
+  <data noupdate="1">
+    <record id="sbc_translation.nas_share_name" model="ir.config_parameter">
+      <field name="key">sbc_translation.nas_share_name</field>
+      <field name="value">GP</field>
+    </record>
+    
+    <record id="sbc_translation.nas_letters_store_path" model="ir.config_parameter">

```

Comment:
You don't need to put module name in xml record_id. `nas_letters_store_path` is enough.